src/initialize/checkpoint.sh

```
@@ -0,0 +1,62 @@
+# checkpoint.sh
+
+_checkpoint_add()
+{
+	local type="$1" msg="$2"
+
+	if [[ -n "$DEBUG" ]]; then
+		local time=$(date +%s)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)